orca-core/src/main/groovy/com/netflix/spinnaker/orca/pipeline/PipelineStarter.groovy

```diff
@@ -54,6 +54,7 @@ class PipelineStarter extends ExecutionStarter<Pipeline> {
       .withAppConfig((Map<String, Serializable>) config.appConfig)
       .withParallel(config.parallel as Boolean)
       .withLimitConcurrent(config.limitConcurrent as Boolean)
+      .withLimitWaiting(config.limitWaiting == null ? true : config.limitWaiting as Boolean)
```

Comment:
agree. 
in the latest code, i ended up not needing the null check due to changing of the variable name/meaning per tomas' suggestion. so i think we are good here.